compiler/src/openxla/compiler/nvgpu/Dialect/CUDNN/Conversion/ConvertCUDNNToRuntime.cpp

```diff
@@ -202,6 +294,7 @@ void populateCudnnToRuntimePatterns(mlir::TypeConverter &typeConverter,
   auto api = std::make_shared<CudnnAPI>();
   patterns.insert<ConvertCudnnGraphOp>(typeConverter, ctx, api);
   patterns.insert<ConvertCudnnReturnOp>(typeConverter, ctx, api);
+  patterns.insert<ConvertCudnnConvolutionOp>(typeConverter, ctx, api);
```

Comment:
I noticed that we aren't consistent with camelCase vs. snake_case. Which one should it be?

Reply:
In `compiler` it should be LLVM style with `camelCase` and under `runtime` a google style with `snake_case`, this is my understanding based on looking at IREE code base, but it's not enforced by a linter, so I keep forgetting about it